cmd/invoke.go

```diff
@@ -375,5 +382,17 @@ func (c invokeConfig) prompt(client *fn.Client) (invokeConfig, error) {
 		return c, err
 	}
 
+	qs = []*survey.Question{
+		{
+			Name: "Insecure",
+			Prompt: &survey.Confirm{
+				Message: "Allow insecure server connections when using SSL",
+				Default: false,
```

Comment:
The default could be set to the current value of `cfg.Insecure` rather than `false` such that a call to `--confirm` which also includes `--insecure` will set the confirm prompt appropriately.
```shell
func invoke --insecure --confirm
```
... would then prompt but with the default set to `true`

Reply:
Right!

Reply:
Fixed